packages/datatrak-web-server/src/app/middleware/attachAccessPolicy.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import { RequestHandler } from 'express';
+import { AccessPolicyBuilder, mergeAccessPolicies } from '@tupaia/auth';
+import { AccessPolicy } from '@tupaia/access-policy';
+
+const { API_CLIENT_NAME } = process.env;
```

Comment:
I always find it nice to use `requireEnv` from `@tupaia/utils` in these kinds of scenarios. It gives a really clear error message if the .env var is missing